### ISSUE TITLE
Help! Why can't login on PD enviroment?

### ISSUE THREAD

**Issue by WayneSunJazz** [WowLegacyCore/HermesProxy#303]
(Enviroment:M1 mac mini + PD + Win11)

A few months ago, I can't open 1.14.0 client by Arctium, so I give up.

Today, I try again, it's opend! But I found I can't login.

<img width="1627" alt="截屏2023-04-28 15 47 46" src="https://user-images.githubusercontent.com/119514624/235092301-3cfadc61-22c4-481b-a368-6a2c43399215.png">
<img width="1628" alt="截屏2023-04-28 15 49 59" src="https://user-images.githubusercontent.com/119514624/235092336-b88cd92b-cfdf-4fa6-b4c7-3f48154f2394.png">

Any suggestion? I'm so closed to achieve playing 1.14 on M1 mac!

**Comment by WayneSunJazz** [comment]
> 

Thank you for reply.

I upload tow pictures, It show Arctium did open the client, but can't login account.

I will try Winterspring, thanks!

**Comment by Fearyii** [comment]
> > I don't think Arctium works with Windows arm64.
> > If you're wanting to play on an M1 Mac, use the Winterspring launcher. It will let you play using the official Mac client: https://github.com/0blu/WinterspringLauncher/releases/download/1.4.0/WinterspringLauncher-macos-arm64.dmg
> 
> Thank you for reply.
> 
> I upload tow pictures, It show Arctium did open the client, but can't login account.
> 
> I will try Winterspring, thanks!

Have u set the Realmlist/Portal to 127.0.0.1 in the WTF folder, becuase if not it wont work no matter what

**Comment by WayneSunJazz** [comment]
> > > I don't think Arctium works with Windows arm64.
> > > If you're wanting to play on an M1 Mac, use the Winterspring launcher. It will let you play using the official Mac client: https://github.com/0blu/WinterspringLauncher/releases/download/1.4.0/WinterspringLauncher-macos-arm64.dmg
> > 
> > 
> > Thank you for reply.
> > I upload tow pictures, It show Arctium did open the client, but can't login account.
> > I will try Winterspring, thanks!
> 
> Have u set the Realmlist/Portal to 127.0.0.1 in the WTF folder, becuase if not it wont work no matter what

Yes,it was.  Same setting, I can login on windows PC. Why doesn't hermesProxy work on PD, help me! 
![截屏2023-04-28 18 31 02](https://user-images.githubusercontent.com/119514624/235124993-7288f3ff-5890-4baf-8fa5-56777450e83d.jpg)

**Comment by WayneSunJazz** [comment]
> Your config seems to be correct. It appears that you are also using _arctium wow launcher_ because we can see a connection attempt in HermesProxy's logs.
> 
> Maybe there is a bug in the chinese 1.14.2 version. Can you try 1.14.0 again? Do you know other chinese players that are using this client?
> 
> Btw. are you using an Apple Silicon Mac or Intel one?

Yes, I use M1 mac mini and PD. 

But since henhouse told me there is winterspring for natviely supporting MacOS M1, so I tried it.

After some issue, I've finally done. winterspring successfully opened 1.14.0 and login everlook.org. And it looks great!

But what I'm really looking for is to play 1.14.0 client on MacOS M1 with single player server Mangos.

Now, the thing is Mangos only has windows version, and I set it all up on PD Win11, but How can I connect mac winterspring to Win11 mangos?.... pretty weird asking hah.

Otherwise, There are many chinese players who enjoy you guys' work very much. As I know, many people play 1.14.0 or 1.14.2 on windows. It works just fine.

And one last qustion, How can I add 1.14.2 in winterspring? I don't see there is a manual switch instruction. Will you natively support?

**Comment by henhouse** [comment]
There are a lot of questions here, and most do not have anything to do with this project (HermesProxy), but I will answer them anyway to hopefully point you in the right directions to where you can get further assistance.

Both vmangos and cmangos support compiling their codebases on an M1 Mac. I assume this "single player server" is a pre-compiled file that someone has provided? If so, then I am guessing they only provided Windows binaries, so yes, you would need to run that within Parallels. You can still connect to it from the host machine (macOS) by editing your `HermesProxy.config` file `ServerAddress` field to point to the LAN IP address of the Windows (Parallels) instance.

Currently you can only use HermesProxy on an Apple Silicon-based Mac via Winterspring and only with 1.14.0 due to how the game must be patched in order to allow connects to Hermes. Windows can be used with 1.14.0, 1.14.1, and 1.14.2, but not macOS at this time.

Since this place is not really where to ask for support for MaNGOS related things, I would suggest joining either the [cmangos](https://discord.gg/Dgzerzb) or [vmangos](https://discord.gg/x9a2jt7) Discord servers and using their support channels to see if community members can provide help with connecting to a local server if you wish to play alone, or use this single player repack project.

**Comment by WayneSunJazz** [comment]
> compiling

Your reply is so professional and helpful. 

"single player server" does mean that I wanna play the game myself on my local server. So I can test, I can have fun, whatever I want. The compiled server now I'm using is called Vanilla bropack v21.

This issue post is now pretty much solved.

Thank you all. best wish.

**Comment by WayneSunJazz** [comment]
> There are a lot of questions here, and most do not have anything to do with this project (HermesProxy), but I will answer them anyway to hopefully point you in the right directions to where you can get further assistance.
> 
> Both vmangos and cmangos support compiling their codebases on an M1 Mac. I assume this "single player server" is a pre-compiled file that someone has provided? If so, then I am guessing they only provided Windows binaries, so yes, you would need to run that within Parallels. You can still connect to it from the host machine (macOS) by editing your `HermesProxy.config` file `ServerAddress` field to point to the LAN IP address of the Windows (Parallels) instance.
> 
> Currently you can only use HermesProxy on an Apple Silicon-based Mac via Winterspring and only with 1.14.0 due to how the game must be patched in order to allow connects to Hermes. Windows can be used with 1.14.0, 1.14.1, and 1.14.2, but not macOS at this time.
> 
> Since this place is not really where to ask for support for MaNGOS related things, I would suggest joining either the [cmangos](https://discord.gg/Dgzerzb) or [vmangos](https://discord.gg/x9a2jt7) Discord servers and using their support channels to see if community members can provide help with connecting to a local server if you wish to play alone, or use this single player repack project.

Your reply is so professional and helpful.

"single player server" does mean that I wanna play the game myself on my local server. So I can test, I can have fun, whatever I want. The compiled server now I'm using is called Vanilla bropack v21.

This issue post is now pretty much solved.

Thank you all. best wishes!

**Comment by WayneSunJazz** [comment]
Add：

Now I complished playing 1.14.0 through winterspring on M1 mac natively with Vanilla bropack v21 local Magons server which is runing on PD Win11.

And I can connect to any private server on internet if I want.
